vllm/distributed/device_communicators/shm_broadcast.py

```diff
@@ -496,6 +463,7 @@ def enqueue(self, obj, timeout: Optional[float] = None):
                 with self.acquire_write(timeout) as buf:
                     buf[0] = 0  # not overflow
                     buf[1:len(serialized_obj) + 1] = serialized_obj
+                self.local_socket.send(b"\x00")
```

Comment:
I remember that during `execute_model`, it calls `broadcast_tensor_dict` to broadcast the tensor. I’m not sure if this affects anything.